client/src/utils/jobsUtil.js

```diff
@@ -128,6 +128,8 @@ export function getJobDisplayInformation(job, app) {
         display.processorsPerNode = job.processorsPerNode;
         display.nodeCount = job.nodeCount;
       }
+
+      display.interactive = app.definition.tags.includes('Interactive');
```

Comment:
Maybe we should expand this to also include "VNC" and "DCV" tags - I can't say for sure if we add "Interactive" to all interactive apps